### PR TITLE
Make it more obvious LDAPS is supported (#460)

### LINKED ISSUE #460: Ability to use ldap over SSL/TLS
I scoured the documents and issues but couldn't see any information about SSL/TLS.

Is there either support for it, a work around or helper proxy or any planned support for it?

I know in most cases it's probably running inside a docker overlay network that's secure, but I'm just considering opening it up to other machines on the network.

Thanks, I love the software, it's so slick and easy to use.

### PR DESCRIPTION
**Pull request by mitchins**
Update example compose config showing both port and alluding to the environment variable that controls the certificates
This will resolve #460 in that searching for "SSL" or "CERT" or "KEY" in the readme yields no results for those not familiar to search for "LDAPS".

It is hoped that anyone reading the main page will easily see that "ssl" (tls) or "certificates" are indeed supported.

If this needs updating or seems redundant, just let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +133,9 @@ services:
       - LLDAP_JWT_SECRET=REPLACE_WITH_RANDOM
       - LLDAP_LDAP_USER_PASS=REPLACE_WITH_PASSWORD
       - LLDAP_LDAP_BASE_DN=dc=example,dc=com
+      # If using LDAPS, configure your cert and keys
+      # - LLDAP_LDAPS_OPTIONS__CERT_FILE=PATH_TO_CERT
```
Comment (nitnelave) [inline README.md:137]: Maybe we can add the file extension? I sometimes get confused between the cert, pem and key file (well, mostly about the pem file, whether it's a cert of key)

**Comment by nitnelave** [comment]
Hey, quick check, do you want to update this PR, or should I take it over?

File: README.md (hunk 1 above)
Comment (martadinata666) [reply README.md]: Im taking over this, and will re-request review

**Review by nitnelave — APPROVED** [review]
Thanks!

**Comment by nitnelave** [comment]
Thank you everyone for the contribution!
This feature is now available in the latest release, [v0.5.0](https://github.com/lldap/lldap/releases/tag/v0.5.0).
You can support LLDAP by starring our repo, contributing some configuration examples and becoming a sponsor.

**Pull request merged**